easybuild/easyconfigs/g/GMP/GMP-5.0.5-intel-2015a.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '5.0.5'
+
+homepage = 'http://gmplib.org/'
+description = """GMP is a free library for arbitrary precision arithmetic, 
+ operating on signed integers, rational numbers, and floating point numbers. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [GNU_SOURCE]
+
+runtest = 'check'
```

Comment:
test report is coming; but yes, it works

For GMP 6.x we should just installed with `GCC/4.9.2` as a toolchain?